view/source/scss/docs.scss

```diff
@@ -864,7 +863,18 @@ div.gallery-popup img {
         &:hover {
             background-color: #999;
         }
+    }
+
+    a.button.large.expand.docsintro {
+        flex-basis: 100%;
+    }
 
+    a.button.large.docsintro {
+        flex-basis: calc(50% - 0.5em);
```

Comment:
See, this is why Gawain isn't allowed to do visual work :stuck_out_tongue_winking_eye: